frontend/app/dashboard/template.hbs

```diff
@@ -23,7 +23,7 @@
                         {{/if}}
                         <p data-test-project-total>{{format-duration p.totalTime}}{{t 'dashboard.remaining'}}</p>
                       </div>
-                      {{svg-battery percentage=p.percentage data-test-percentage=true}}
+                      {{timed-battery percentage=p.percentage index=index data-test-percentage=true class="uk-flex uk-flex-center"}}
```

Comment:
[add the classes in the component](https://www.emberjs.com/api/ember/3.3/classes/Component/properties/classNames?anchor=classNames) if they are not dependant on any content of this template.: